tests/test_playwright.py

```diff
@@ -0,0 +1,262 @@
+import os
+import pytest
+
+from playwright.sync_api import Page, expect
+from helpers.postgres_helper import (
+    User,
+    create_user_using_psql,
+    execute_postgres_query,
+    fetch_postgres_query,
+    follow_user_using_psql,
+    tweet_using_psql,
+)
+from helpers.test_helper import (
+    BASE_URL,
+    generate_random_email,
+    generate_random_password,
+    generate_random_username,
+)
+
+MY_TIMELINE_URL = f"{BASE_URL}/"
+PUBLIC_TIMELINE_URL = f"{BASE_URL}/public"
+SIGN_UP_URL = f"{BASE_URL}/register"
+SIGN_IN_URL = f"{BASE_URL}/login"
+SIGN_OUT_URL = f"{BASE_URL}/logout"
+USER_TIMELINE_URL = f"{BASE_URL}/user"
+
+
+def sign_in_using_playwrght(page: Page, useTestUser=True):
+    if not useTestUser:
+        page.goto(SIGN_UP_URL)
+        global username, email, password
+
+        username = generate_random_username()
+        email = generate_random_email()
+        password = generate_random_password()
+
+        page.locator("input[name='username']").fill(username)
+        page.locator("input[name='email']").fill(email)
+        page.locator("input[name='password']").fill(password)
+        page.locator("input[name='password2']").fill(password)
+        page.locator("input[type='submit']").click()
+
+    page.goto(SIGN_IN_URL)
+
+    page.locator("input[name='username']").fill(
+        test_username if useTestUser else username
+    )
+    page.locator("input[name='password']").fill(
+        test_password if useTestUser else password
+    )
+    page.locator("input[type='submit']").click()
+
+
+def sign_in_user_using_playwright(page: Page, user: User):
+    page.goto(SIGN_IN_URL)
+
+    page.locator("input[name='username']").fill(user.username)
+    page.locator("input[name='password']").fill(user.password)
+    page.locator("input[type='submit']").click()
+
+
+test_username = "test"
+test_email = "test@test.com"
```

Comment:
Vi bruger vist "...@test.test" til testbrugere, bare for ocd giver det nok mening at lave .test her også
```suggestion
test_email = "test@test.test"
```